server.go

```diff
@@ -117,19 +87,41 @@ func CheckBlacklistHandler(c echo.Context) error {
 
 	log.Println(msg)
 
-	return c.JSON(http.StatusCreated, msg)
+	return c.JSON(http.StatusCreated, returnMessage{Message: msg})
 
 }
 
+// Check and create db onetime only, when proram starts.
+func initDB() error {
+	return db.Update(func(tx *bolt.Tx) error {
+		_, err := tx.CreateBucketIfNotExists(dbName)
+		return err
+	}
+}
+
 func main() {
 
+	/* NOTE: This did not work because shorthand declaration is local to the closet syntatic block {..} 
```

Comment:
ah that short variable declarations :open_hands: 

https://golang.org/ref/spec#Short_variable_declarationsl